conf/production.ini

```diff
@@ -9,11 +9,6 @@ use: call:h.app:create_app
 [filter:proxy-prefix]
 use: egg:PasteDeploy#prefix
 
-[server:main]
-use: egg:gunicorn#main
-bind: unix:/tmp/gunicorn-web.sock
-proc_name: web
```

Comment:
Renamed `app.ini` to `production.ini` as in our other apps, and again moved the `[server:main]` section into `gunicorn.conf.py`. Again the `proc_name` setting was unnecessary here.